packages/network/src/v2/fetchStoreEvents.ts

```diff
@@ -6,18 +6,18 @@ import { storeEvents } from "./common";
 import { ecsEventFromLog } from "./ecsEventFromLog";
 
 export async function fetchStoreEvents(
-  contract: Contract,
+  world: Contract,
```

Comment:
should this be `store`? I'm assuming it works with all contracts implementing the `IStore` interface, not just `world`?

Reply:
yep, feel free to make that change! I think I called it world because that's what the sync worker was using